daemon/oci_windows.go

```diff
@@ -344,6 +285,82 @@ func (daemon *Daemon) createSpecWindowsFields(c *container.Container, s *specs.S
 	return nil
 }
 
+var errInvalidCredentialSpecSecOpt = fmt.Errorf("invalid credential spec security option - value must be prefixed by 'file://', 'registry://', or 'base64://' followed by a non-empty value")
```

Comment:
I have a PR in the works (#38770) to normalize all of the error values to use the `errdefs` definitions. Can you modify this and any other _new_ errors to use those error types, so that I don't have to repeat the work when rebasing that PR?

Reply:
Sure thing, doing that today or tomorrow.

Reply:
Done :)